applications/luci-app-ddns/htdocs/luci-static/resources/view/ddns/overview.js

```diff
@@ -440,8 +440,8 @@ return view.extend({
 
 		}
 
-		o = s.taboption('global', form.Value, 'cacert', _('CA Certs path'));
-		o.description = _('CA certificates path that will be used to download services data. Set IGNORE to skip certificate validation.');
+		o = s.taboption('global', form.Value, 'cacert', _('CA bundle file path'));
+		o.description = _('File path of the CA bundle that will be used to download services data. Set IGNORE to skip certificate validation.');
```

Comment:
In this case, let's avoid ambiguity of path, and refer directly to the file itself. Please re-word to
```
		o = s.taboption('global', form.Value, 'cacert', _('CA Certs bundle file'));
		o.description = _('CA bundle file used to download services data. Set IGNORE to skip certificate validation.');
```